### PR TITLE
docs(graphql): Complexity on Query and Mutation

### PR DESCRIPTION
**Pull request by wSedlacek**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

Documents this [PR](https://github.com/nestjs/graphql/pull/1088)

### CODE REVIEW

**Comment by kamilmysliwiec** [comment]
Would you mind taking a look real quick? 🙏

**Review by jmcdo29 — APPROVED** [review]

File: content/graphql/complexity.md [hunk 1]
```diff
@@ -84,3 +84,14 @@ Alternatively, you can define the estimator function:
 @Field({ complexity: (options: ComplexityEstimatorArgs) => ... })
 title: string;
 ```
+
+#### Query/Mutation-level complexity
+
+Additionally, `@Query()` and `@Mutation()` may have a `complexity` property specified like so:
```
Comment (kamilmysliwiec) [inline content/graphql/complexity.md:90]:
```suggestion
In addition, `@Query()` and `@Mutation()` decorators may have a `complexity` property specified like so:
```

File: content/graphql/complexity.md [hunk 2]
```diff
@@ -84,3 +84,14 @@ Alternatively, you can define the estimator function:
...
+
+```typescript
+@Query({ complexity: (options: ComplexityEstimatorArgs) => options.args.count * options.childComplexity })
+items(@Args('count') count: number) {
+  return itemsService.getItems({ count });
```
Comment (kamilmysliwiec) [inline content/graphql/complexity.md:95]:
```suggestion
  return this.itemsService.getItems({ count });
```

**Comment by kamilmysliwiec** [comment]
Thanks!

**Pull request merged**